src/test/kotlin/com/terraformation/backend/customer/model/PermissionRequirementsTest.kt

```diff
@@ -708,6 +709,31 @@ internal class PermissionRequirementsTest : RunsAsUser {
   @Test
   fun updateSpecies() = allow { updateSpecies(speciesId) } ifUser { canUpdateSpecies(speciesId) }
 
+  @Test
+  fun updateSpecificGlobalRoles() {
+    assertThrows<AccessDeniedException> {
+      requirements.updateSpecificGlobalRoles(setOf(GlobalRole.SuperAdmin))
+    }
+
+    grant {
+      user.canUpdateSpecificGlobalRoles(
+          setOf(GlobalRole.AcceleratorAdmin, GlobalRole.TFExpert, GlobalRole.ReadOnly))
+    }
+    assertThrows<AccessDeniedException> {
+      requirements.updateSpecificGlobalRoles(setOf(GlobalRole.SuperAdmin))
+    }
+
+    grant { user.canUpdateSpecificGlobalRoles(setOf(GlobalRole.SuperAdmin)) }
+    requirements.updateSpecificGlobalRoles(setOf(GlobalRole.SuperAdmin))
+
+    grant {
+      user.canUpdateSpecificGlobalRoles(
+          setOf(GlobalRole.AcceleratorAdmin, GlobalRole.TFExpert, GlobalRole.ReadOnly))
+    }
+    requirements.updateSpecificGlobalRoles(
+        setOf(GlobalRole.AcceleratorAdmin, GlobalRole.TFExpert, GlobalRole.ReadOnly))
+  }
```

Comment:
I wonder if some of this exercising of the logic for matching against what's in the set of roles belongs in `PermissionTest` instead since it's really testing the logic in `IndividualUser` rather than the logic in `PermissionRequirements`.

Reply:
I moved the tests over to `PermissionTest`